### ISSUE TITLE
Ability to add custom “Other relationship” types

### ISSUE THREAD

**Issue by mgrhm** [monicahq/monica#4689]
It’s really useful to be able to link people by their relationships to one another.

It would be useful to be able to edit the list of relationships like you can with activities and contact types.  This would allow users to add the types they need themselves – in my case, it would be useful to be able to add ‘flatmates’ as a relationship type.

**Comment by LouisRast** [comment]
Second this!

**Comment by LouisRast** [comment]
> It’s really useful to be able to link people by their relationships to one another.
> 
> It would be useful to be able to edit the list of relationships like you can with activities and contact types. This would allow users to add the types they need themselves – in my case, it would be useful to be able to add ‘flatmates’ as a relationship type.

Second this!

**Comment by virtadpt** [comment]
Thirded.  "Trapped someplace by covid-19 quarantine" is a thing, unfortunately.

**Comment by travelg** [comment]
Any progress on this ? 
Thank you.

**Comment by v-marinkov** [comment]
Does anyone have any workaround for this? I would like to use Monica for academic networking, so things like "co-author" would be fantastic.

**Comment by EasternPA** [comment]
Thirding? Fourthing? I really need to add in-laws. Brother-in-law, sister-in-law, mother-in-law, father-in-law. These are all people who become related to you through marriage. I can't register my wife's parents as my in-laws.

**Comment by Poldi1977** [comment]
It would be great if the list would be extended. That should also go hand in hand with relatively little effort. Of course, it would be ideal if you could add entries yourself.

**Comment by beemej** [comment]
Piling on to this one.  Ideally adding your own types of family relationships would be best, but just expanding to include in-laws for the major categories would be a huge benefit.

**Comment by Gabbalo** [comment]
Yes, in-laws would be a great addition to the relationship-list so count this as a push and vote please ;)

**Comment by brandonleon** [comment]
Just adding a comment for one more person interested in this. With may polyamorous friends I'd love to be able to track my friends metamours.

**Comment by Dalarialus** [comment]
I'd also like to see this as a feature!

**Comment by beemej** [comment]
The in-law relationships in particular would be super helpful - they are just so much more common to need than some others (such as god-son and god-daughter!).  I see this request has been made repeatedly.  Is there any intention to tackle it?

**Comment by rayandaod** [comment]
It would be super useful to have this!

**Comment by edo-dzell** [comment]
I’d like to bump this request as well.

For a Personal CRM, mapping extended family is crucial. Currently, the lack of specific "in-law" options is a significant limitation for me.

I am specifically missing:
- Mother-in-law
- Father-in-law
- Son-in-law
- Daughter-in-law

Using generic labels or notes for these close relationships makes organization difficult. It would be great to see these standard family roles added soon.

**Comment by beemej** [comment]
Thanks for the endorsement – I could not agree more.  It is a big limitation and seems like it would be one of the easier fixes.
